conwayste/src/client.rs

```diff
@@ -331,7 +331,11 @@ impl MainState {
         color_settings.cell_colors.insert(CellState::Wall,           Color::new(0.617,  0.55,  0.41, 1.0));
         color_settings.cell_colors.insert(CellState::Fog,            Color::new(0.780, 0.780, 0.780, 1.0));
 
-        let font = graphics::Font::default(); // Provides DejaVuSerif.ttf
+        let mut font = graphics::Font::default(); // Provides DejaVuSerif.ttf
```

Comment:
You should be able to replace this whole section with:
```rust
let font = graphics::Font::default()
       .map_err(|e| GameError::FilesystemError( format!("Could not load or find font. {:?}", e) ))?;
```

Reply:
Ah thanks for the tip!